packages/manager/modules/pci/src/projects/project/training/registries/add/translations/Messages_fr_FR.json

```
@@ -0,0 +1,11 @@
+{
+  "pci_projects_project_training_registries_add_url": "URL du Docker Registry Privé",
```

```suggestion
  "pci_projects_project_training_registries_add_url": "URL du registre Docker Privé",
```
to be consistent with https://github.com/ovh/manager/pull/3982/files#diff-6b028597b7917516094a8a7a74d32d7ee1aa9c0dd4dbbffdd6f0952d56131a2fR2